### PR TITLE
Ports arm and leg guards

### PR DESCRIPTION
**Pull request by Alberyk**
Due to this suggestion: https://forums.aurorastation.org/viewtopic.php?f=21&t=10720
-ports, from polaris, armored leg guards, that come in riot, bulletproof and ablative variations
-ports, from polaris, armored arm guard, that come in riot, bulletproof and ablative variations
-they are also added to the armory

### CODE REVIEW

**Comment by LordFowl** [comment]
This doesn't seem to actually add them?

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -117,7 +117,6 @@
 	desc = "A suit of armor with heavy padding to protect against melee attacks. Looks like it might impair movement."
 	icon_state = "riot"
 	item_state = "swat_suit"
-	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
```
Comment (printer16) [inline code/modules/clothing/suits/armor.dm]: Is removing UPPER_TORSO|LOWER_TORSO intentional?

Comment (Alberyk) [reply code/modules/clothing/suits/armor.dm]: Yes.

**Comment by Alberyk** [comment]
I will come back to this later.

**Pull request closed without merging**